chains/evm/contracts/test/ccvs/CommitteeVerifier/CommitteeVerifierSetup.t.sol

```diff
@@ -14,6 +14,7 @@ contract CommitteeVerifierSetup is BaseVerifierSetup {
   uint256 internal constant PRIVATE_KEY_0 = 0x60b919c82f0b4791a5b7c6a7275970ace1748759ebdaa8a5c3a4b2f5a8b1e8d1;
   address internal constant MOCK_SENDER = 0x3333333333333333333333333333333333333333;
   address internal constant MOCK_RECEIVER = 0x4444444444444444444444444444444444444444;
+  uint64 public constant DEFAULT_SOURCE_CHAIN_SELECTOR = 1;
```

Comment:
nit: SOURCE_CHAIN_SELECTOR exists in BaseTest so it's available in all tests

Reply:
Yes good point! [245dc24](https://github.com/smartcontractkit/chainlink-ccip/pull/1417/commits/245dc24d04acb82b8d79f4545450a93e3216522b)